dashboard/client/src/pages/node/GPUColumn.tsx

```diff
@@ -0,0 +1,51 @@
+import { Typography } from "@material-ui/core";
+import React from "react";
+import { NodeDetail } from "../../type/node";
+import { Worker } from "../../type/worker";
+import {
+  GPU_COL_WIDTH,
+  NodeGPUEntry,
+  WorkerGPUEntry,
+} from "../dashboard/node-info/features/GPU";
+
+export const NodeGPUView = ({ node }: { node: NodeDetail }) => {
+  return (
+    <div style={{ minWidth: GPU_COL_WIDTH }}>
+      {node.gpus !== undefined && node.gpus.length !== 0 ? (
```

Comment:
nit: you could do `node.gpus?.length` instead of `node.gpus !== undefined && node.gpus.length !== 0`

Reply:
Unfortunately, the typescript compiler isn't smart enough to realize `node.gpus` is not null in the next line when node.gpus.length is not null.